kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/core/annotation/concurrency.kt

```diff
@@ -0,0 +1,25 @@
+package io.kotest.core.annotation
+
+/**
+ * When added to a spec, will mark that spec to not run in parallel, regardless
+ * of concurrency settings.
+ */
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class Isolate
```

Comment:
Very nitty but important for the public API:

I know the previous version was called Isolate, but if this is a breaking change I suggest changing this guy to `Isolated`. As `Parallel` in this case is a noun and `Isolate` is more of a verb.

I think it is more sense to have 
- My test is running parallel
- My test is running isolated

Than having
- My test is running parallel
- My test is running isolate

Reply:
It's not a breaking change, the package has stayed the same.